commercial/app/model/commercial/jobs/Job.scala

```diff
@@ -43,6 +45,28 @@ object Job {
     sectorIds = (xml \ "Sectors" \ "Sector") map (_.text.toInt),
     salaryDescription = (xml \ "SalaryDescription").text
   )
+
+  def toJsonTuple(job: Job): Option[(Int, String, String, String, String, Option[String], String)] =
+    Some((
+      job.id,
+      job.title,
+      job.listingUrl,
+      job.recruiterLogoUrl,
+      job.recruiterName,
+      job.locationDescription,
+      job.shortSalaryDescription
+    ))
+
+  implicit val writesJob: Writes[Job] = (
+    (JsPath \ "id").write[Int] and
+    (JsPath \ "title").write[String] and
+    (JsPath \ "listingUrl").write[String] and
+    (JsPath \ "recruiterLogoUrl").write[String] and
+    (JsPath \ "recruiterName").write[String] and
+    (JsPath \ "locationDescription").write[Option[String]] and
+    (JsPath \ "shortSalaryDescription").write[String]
+  )(unlift(Job.toJsonTuple))
```

Comment:
You can also create a `new Writes[Job]` and define inside the `writes(job: Job): JsValue` method like done [here](https://github.com/guardian/frontend/blob/b2527874ac8f94f4d61b1bed2d91f0d930b5c590/admin-jobs/app/models/BreakingNews.scala#L30-L42) so you don't have to define this weird intermediary tuple data structure